src/js/cookieMessage.js

```diff
@@ -30,6 +29,11 @@ class CookieMessage {
 		message.classList.add('o-cookie-message--hidden');
 	}
 
+	static showMessage () {
+		const message = document.querySelector('[data-o-component="o-cookie-message"]');
```

Comment:
you could make this non static and have access to the element so you don´t need to query for it again